packages/Spartan-secq/.github/workflows/rust.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Test Spartan
```

Comment:
I agree that we should set up CI but if we are going to add a GitHub action can we add it in the root dir?

Reply:
Or we can do that later by removing this rust.yml and merging this PR first!

Reply:
ya let's remove this `rust.yml` for now and add CI at the root dir later